steward/src/cellars/adaptors/aave_v2.rs

```diff
@@ -309,3 +311,41 @@ pub(crate) fn aave_debt_token_adaptor_v2_calls(
 
     Ok(calls)
 }
+
+pub(crate) fn aave_v2_enable_asset_as_collateral_adaptor_v1_calls(
```

Comment:
Like the Uniswap case, this is a separate adaptor so we may want a separate file.